src/screen.rs

```diff
@@ -92,6 +92,7 @@ pub struct Screen {
     panes_to_hide: HashSet<RawFd>,
     active_terminal: Option<RawFd>,
     os_api: Box<dyn OsApi>,
+    is_fullscreen: bool,
```

Comment:
Can we change the name to fullscreen_is_active? I feel it'll be a little clearer.

Reply:
agreed. I am not so good with names XD.